bluepyopt/deapext/utils.py

```diff
@@ -83,6 +83,23 @@ def __init__(self, *args, **kwargs):
         super(WSListIndividual, self).__init__(*args, **kwargs)
 
 
+def generate_starters(ind_size, lambda_, clip=True, clip_threshold=0.8):
+    """Generate a new set of starter individual uniformaly distributed in a
+    hypercube centered at zero and of length 2. * clip_threshold. Used by SO
+    and MO-CMA"""
+
+    ubound = clip_threshold if clip else 1.
```

Comment:
Why do we initialize it between -1 and 1? Why not between parameter bounds?

Reply:
The parameter space used by CMA is normalized